testador/missing.sh

```diff
@@ -4,14 +4,8 @@
 # Go to tests folder
 cd $(dirname "$0")
 
-# List of all functions
-all=$(ls -1 ../zz/ | sed 's/\.sh$//; s|.*/||' | sort)
-
-# List of already tested functions
-tested=$(ls -1 zz*.sh | sed 's/\.sh$//' | sort)
-
-# Who's not tested?
-for zz_func in $all
+# All zz functions have a counterpart in the test directory?
+for zz_func in ../zz/*
 do
-	echo "$tested" | grep "^$zz_func$" > /dev/null || echo "$zz_func"
+	test -f $(basename $zz_func) || echo "$zz_func"
```

Comment:
E se no lugar de ` echo "zz_func" ` for ` echo "${zz_func##*/}" `?

Reply:
Arrumado, agora mostra só o nome da função, sem path